lib/enumerable/helpers.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def check_valid(input, parameter)
+    return input.is_a?(parameter) if parameter.is_a? Class
+
+    if parameter.is_a? Regexp
+      return false if input.is_a?(Numeric)
+
+      return parameter.match(input)
+    end
+    (input == parameter)
+  end
+
+  def verify_input(initial)
+    start = initial.nil? ? 1 : 0
+    start
+  end
+
+  def multiply_els(arr, multiplier)
+    arr.my_inject(multiplier) { |memo, val| memo * val }
+  end
```

Comment:
Let's move `multiple_else` out of the `Enumerable` module.

We should be able to call `multiple_else` with an array as an argument only.

> Test your `#my_inject` by creating a method called `#multiply_els` which multiplies all the elements of the array together by using `#my_inject`, e.g. `multiply_els([2,4,5]) #=> 40`

Reply:
Done!
![image](https://user-images.githubusercontent.com/36519478/68699549-4d901700-0551-11ea-9e8d-b1c9f3c61bb2.png)